### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by tomijaga**

### CODE REVIEW

File: pages/dashboard.tsx [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+  ];
+
+  return (
+    <>
+      <Layout>
+        <div className="font-mono py-12 px-12 grid bg-gray-100 md:grid-cols-4 md:grid-rows-5 grid-cols-1 gap-4">
+          <div className="text-sm p-6 bg-white text-primary shadow-md md:row-start-1 md:row-span-5 flex flex-col space-y-4">
+            <div className="text-center flex flex-col  items-center w-full">
+              <img
+                src="http://tinygraphs.com/labs/isogrids/hexa16/TNB Crow?theme=seascape&numcolors=4&size=220&fmt=svg"
```
Comment (denistsoi) [inline pages/dashboard.tsx:268]: use https:// where possible

**Review by denistsoi — CHANGES_REQUESTED** [review]
change http to https thx

File: pages/dashboard.tsx [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+          <Button type="text" disabled={page >= trades.length / 10}>
+            Next
+          </Button>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export default function dashboard() {
```
Comment (denistsoi) [inline pages/dashboard.tsx:167]: refer to FetchTrades on https://github.com/tnbCrow/Frontend/blob/main/pages/index.tsx#L190

File: pages/dashboard.tsx [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+            <Button>Wallet</Button>
+
+            <div className="h-full flex flex-col justify-between">
+              <div className="md:space-y-8 md:mt-8 flex md:flex-col justify-around flex-row mb-4">
+                <div className="text-left space-y-1 flex flex-col m sm:w-auto w-28">
+                  <h4>
+                    <b>Verification</b>
+                  </h4>
+                  <span>
+                    Phone number <b>verified</b>
```
Comment (denistsoi) [inline pages/dashboard.tsx:288]: for bold, you can use className="font-bold"

File: pages/login.tsx [hunk 4]
```diff
@@ -1,7 +1,7 @@
-import Head from "next/head"
-import Link from "next/link"
-import { FormButton, FormCard, FormInput } from "@components/FormComponents"
-import { Layout } from "@components/Layout"
+import Head from "next/head";
+import Link from "next/link";
```
Comment (denistsoi) [inline pages/login.tsx:2]: perhaps we have different linting/save preferences (i tend not to use semi colons)

**Review by denistsoi — CHANGES_REQUESTED** [review]

File: pages/dashboard.tsx (hunk 1 above)
Comment (tomijaga) [reply pages/dashboard.tsx]: I changed it but they don't support https.

File: pages/dashboard.tsx (hunk 3 above)
Comment (tomijaga) [reply pages/dashboard.tsx]: Thanks

File: pages/login.tsx (hunk 4 above)
Comment (tomijaga) [reply pages/login.tsx]: Yh we do. I added a `.prettierrc` config file so the project can have the same linting seetings

File: pages/dashboard.tsx (hunk 2 above)
Comment (tomijaga) [reply pages/dashboard.tsx:242]: I changed it to reflect the api 
```ts
export interface Trade {
  created_at: string
  uuid: string
  buyer: string
  seller: string
  agent: string
  rate: number
  amount: number
  payment: PaymentOptions
}
```

**Comment by tomijaga** [comment]
image

Hey @denistsoi , I keep getting this error where the buyer is sometimes red and the seller is sometimes green and i don't know the source of it

```Warning: Prop `className` did not match. Server: "px-6 py-4 whitespace-nowrap text-sm uppercase text-green-500 " Client: "px-6 py-4 whitespace-nowrap text-sm uppercase text-red-500 "```


```html
<td className={`px-6 py-4 whitespace-nowrap text-sm uppercase ${
      role === 'buyer'
         ? 'text-green-500'
         : 'text-red-500'
   } `}
>
   {role}
</td>
```

**Comment by denistsoi** [comment]
There's two things happening -

1. You're trying to generate a random collection of mock data when the component is mounted (without using useEffect).
2. The error is related to the data doesn't match to the server, therefore you can explicitly state that the data should be passed as a `getStaticProps` or you generate the data before the application (e.g. a JSON file).

**Comment by tomijaga** [comment]
@denistsoi Thanks for the help. The text colours are properly displayed now

**Pull request merged**